src/test/java/com/google/sps/UserUtilTest.java

```diff
@@ -159,4 +160,105 @@ public void nullUserEntityTitle(){
       Assert.assertEquals(expected, actual);
   }
 
+    // Given an empty string for game id, addGameToUser should return false
+  @Test
```

Comment:
Hannah mentioned that for comments explaining test, you dont need to put what you except the outcome to be you can just say what it is testing. For instance here it would be "Test empty string as paramter for game id" or something similar

Reply:
got it, I changed all of the comments to reflect that